scripts/setup_zipkin.sh

```diff
@@ -24,7 +24,7 @@
 
 set -e
 # install zipkin pods
-kubectl apply -f https://raw.githubusercontent.com/istio/istio/release-1.12/samples/addons/extras/zipkin.yaml
+kubectl apply -f https://raw.githubusercontent.com/istio/istio/release-1.16.0/samples/addons/extras/zipkin.yaml
```

Comment:
This file does not exist.
https://github.com/vhive-serverless/vHive/actions/runs/4222257782/jobs/7330544761#step:9:481